src/Glennsl__BsBenchmarkJs.re

```diff
@@ -0,0 +1,971 @@
+type benchmark;
+type suite;
+type suiteOptions;
+type deferred;
+type event;
+type options;
+type times;
+type stats;
+type platform;
+type support;
+type testFn = (. unit) => unit;
+type setupFn = (. unit) => unit;
+type teardownFn = (. unit) => unit;
+type eventHandler = (. event) => unit;
+type eventType = [ | `abort | `complete | `cycle | `error | `reset | `start | `unknown];
+
+let eventTypeFromString: string => eventType =
+  fun
+  | "abort" => `abort
+  | "complete" => `complete
+  | "cycle" => `cycle
+  | "error" => `error
+  | "reset" => `reset
+  | "start" => `start
+  | _ => `unknown;
+
+let eventTypeToString: eventType => string =
+  fun
+  | `abort => "abort"
+  | `complete => "complete"
+  | `cycle => "cycle"
+  | `error => "error"
+  | `reset => "reset"
+  | `start => "start"
+  | `unknown => "";
```

Comment:
Right, that was my initial design. I suppose it will work now, but at the time, I was having trouble getting the `@bs.string` attribute to work when specifying a module type or `.rei` file. I've since removed the module type definitions. I can go back and fix that now.